Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -143,7 +145,7 @@ public virtual void Initialize(IAlgorithm algorithm, IBrokerage brokerage, IResu
             {
                 throw new ArgumentNullException("brokerage");
             }
-
+            _orderRequestQueue = new BusyBlockingCollection<OrderRequest>();
```

Comment:
maybe a note saying why this impl uses the thread approach -- I'm assuming it's because we want live algorithms to have this run on a separate thread as is the current behavior -- but for the future generations :)